src/client/cockpit-client

```diff
@@ -30,9 +30,14 @@ import tempfile
 import textwrap
 
 gi.require_version("Gtk", "3.0")  # NOQA
-gi.require_version("WebKit2", "4.0")  # NOQA
+from gi.repository import GLib, Gio, Gtk  # NOQA
 
-from gi.repository import GLib, Gio, Gtk, WebKit2  # NOQA: E402
+try:
+    gi.require_version("WebKit2", "4.1")  # NOQA
```

Comment:
I think you should be able to drop the `# NOQA` since this is no longer a top-level import, no?

Reply:
Indeed, thanks! Fixed.